pkg/pipelines/eventlisteners/eventlisteners.go

```diff
@@ -1,23 +1,28 @@
 package eventlisteners
 
 import (
+	"fmt"
+
 	triggersv1 "github.com/tektoncd/triggers/pkg/apis/triggers/v1alpha1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // Filters for interceptors
 const (
-	devCIBuildFilters = "(header.match('X-GitHub-Event', 'pull_request') && body.action == 'opened' || body.action == 'synchronize') && body.pull_request.head.repo.full_name == 'GITHUB_REPO'"
+	devCIBuildFilters = "(header.match('X-GitHub-Event', 'pull_request') && body.action == 'opened' || body.action == 'synchronize') && body.pull_request.head.repo.full_name == '%s'"
 
-	decCDDeployFilters = "(header.match('X-GitHub-Event', 'push') && body.repository.full_name == 'GITHUB_REPO') && body.ref.startsWith('refs/heads/master')"
+	devCDDeployFilters = "(header.match('X-GitHub-Event', 'push') && body.repository.full_name == '%s') && body.ref.startsWith('refs/heads/master')"
 
-	stageCIDryRunFilters = "(header.match('X-GitHub-Event', 'pull_request') && body.action == 'opened' || body.action == 'synchronize') && body.pull_request.head.repo.full_name == 'GITHUB_STAGE_REPO'"
+	stageCIDryRunFilters = "(header.match('X-GitHub-Event', 'pull_request') && body.action == 'opened' || body.action == 'synchronize') && body.pull_request.head.repo.full_name == '%s'"
 
-	stageCDDeployFilters = "(header.match('X-GitHub-Event', 'push') && body.repository.full_name == 'GITHUB_STAGE_REPO') && body.ref.startsWith('refs/heads/master')"
+	stageCDDeployFilters = "(header.match('X-GitHub-Event', 'push') && body.repository.full_name == '%s') && body.ref.startsWith('refs/heads/master')"
 )
 
 // GenerateEventListener will create the required eventlisteners
-func GenerateEventListener() triggersv1.EventListener {
+func GenerateEventListener(baseRepo string) triggersv1.EventListener {
+	githubRepo := baseRepo
+	githubStageRepo := baseRepo + "-stage-config"
```

Comment:
Looks good, you could rename the parameter to `githubRepo` which would drop the need to have the copy?

Reply:
Ah..Yes. Renaming the parameter would drop the copy. I'll just rename it.